sdk/core/azure-core/azure/core/pipeline/transport/aiohttp.py

```diff
@@ -90,7 +90,8 @@ def __init__(self, *, session=None, loop=None, session_owner=True, **kwargs):
         if not self.session and self._session_owner:
             self.session = aiohttp.ClientSession(
                 loop=self._loop,
-                trust_env=self._use_env_settings
+                trust_env=self._use_env_settings,
+                skip_auto_headers=['Content-Type']
             )
```

Comment:
This change has already been merged with master - so you can remove it from here and just pull the latest updates :)